lib/rspec/rails/matchers/action_cable.rb

```diff
@@ -50,7 +50,13 @@ def have_broadcasted_to(target = nil)
 
         ActionCable::HaveBroadcastedTo.new(target, channel: described_class)
       end
-      alias_method :broadcast_to, :have_broadcasted_to
+
+      def broadcast_to(target = nil)
+        RSpec::Matchers::AliasedMatcher.new(
+          have_broadcasted_to(target),
+          ->(old_desc) { old_desc.gsub("have broadcasted", "broadcast") }
+        )
+      end
```

Comment:
You can't use that class here, its a private api of rspec-expectations, instead:

```suggestion
      alias_matcher :broadcast_to, :have_broadcasted_to { |desc| desc.gsub("have broadcasted", "broadcast") }
```

You may need to ensure the matcher DSL is included which is fine to do.